tests/tests_common_test.go

```diff
@@ -117,3 +119,15 @@ func isStatusDeployed(obj *v1alpha1.SSP) bool {
 		progressing.Status == core.ConditionFalse &&
 		degraded.Status == core.ConditionFalse
 }
+
+func CanIAccess(sar *authorization.SubjectAccessReview) (bool, error) {
```

Comment:
This function can be inlined. It only calls another function and returns its result directly.